docs/topics/Gateway.md

```diff
@@ -1053,6 +1058,40 @@ Sent when a guild role is deleted.
 | guild_id | snowflake | id of the guild |
 | role_id  | snowflake | id of the role  |
 
+### Guild Scheduled Event Create
+
+Sent when a guild event is created. The inner payload is a [Guild Event](#DOCS_RESOURCES_GUILD_EVENT/guild-event-object) object.
+
+### Guild Scheduled Event Update
+
+Sent when a guild event is updated. The inner payload is a [Guild Event](#DOCS_RESOURCES_GUILD_EVENT/guild-event-object) object.
+
+### Guild Scheduled Event Delete
+
+Sent when a guild event is deleted. The inner payload is a [Guild Event](#DOCS_RESOURCES_GUILD_EVENT/guild-event-object) object. See [Guild Scheduled Event Status](#DOCS_RESOURCES_GUILD_EVENT/guild-event-object-guild-scheduled-event-status) to determine how the event was deleted.
+
+### Guild Scheduled Event User Create
+
+Sent when a user has subscribed to a Guild Event.
+
+###### Guild Scheduled Event User Create Event Fields
+
+| Field                    | Type      | Description           |
+| ------------------------ | --------- | --------------------- |
+| guild_scheduled_event_id | snowflake | id of the guild event |
+| user_id                  | snowflake | id of the user        |
```

Comment:
A `guild_id` field would have been really helpful in the payload of these two `GUILD_SCHEDULED_USER_*` events

Reply:
@msciotti Any thoughts?

Reply:
we've changed it to include `guild_id` now :) but the event names will be changed. I'll explain in a separate comment